terraform/modules/jenkins/elb.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_elb" "jenkins_elb" {
+  name            = "Jenkins-ELB"
+  subnets         = ["${aws_instance.jenkins3.subnet_id}"]
+  instances       = ["${aws_instance.jenkins3.id}"]
+  security_groups = ["${aws_security_group.jenkins_elb_security_group.id}"]
+
+  listener {
+    instance_port      = 80
+    instance_protocol  = "http"
+    lb_port            = 443
+    lb_protocol        = "https"
+    ssl_certificate_id = "${aws_acm_certificate.jenkins_elb_certificate.arn}"
+  }
+
+  listener {
+    instance_port     = 22
+    instance_protocol = "tcp"
+    lb_port           = 22
+    lb_protocol       = "tcp"
+  }
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = 5
+    target              = "TCP:22"
```

Comment:
I guess the thought is that as long as we have SSH access we can take a peek at what's going wrong the the web interface, but not having SSH we're stuffed and need a reboot?

Reply:
It was more than when bringing up a new, clean instance, nothing is listening on port 80 as nginx isn't there, so there'll be no 200 OK. ssh is there though.